react/__mocks__/vtex.render-runtime.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export const ExtensionPoint = ({ children, id }) => (
+  <div className={`ExtensionPoint-mock-${id}`}>{children}</div>
+)
+
+export const withRuntimeContext = Comp => {
+  const runtime = { account: 'account', culture: { country: 'country' } }
+  return props => <Comp runtime={runtime} {...props} />
+}
+export const NoSSR = ({ children }) => children
```

Comment:
Could you wrap those children on a <div>?

```suggestion
export const NoSSR = ({ children }) => <div className="nossr-mock"children</div>
```